_config.yml

```diff
@@ -95,7 +95,7 @@ defaults:
   - scope:
       path: "install"
     values:
-      win_latest_build: "docker-17.06.2-ee-17"
+      win_latest_build: "docker-18.09.0"
```

Comment:
Looks like we've released `18.09.1` according to https://download.docker.com/components/engine/windows-server/index.json. Should we update to that version?

Reply:
The only 18.09.1 package is a release candidate "18.09.1-rc1", there is not a stable GA release yet of 18.09.1 :)